back/spec/acceptance/project_custom_fields_spec.rb

```diff
@@ -103,25 +73,12 @@
     end
   end
 
-  describe 'in a timeline project with an active ideation phase with custom fields' do
-    let(:project) { create(:project_with_active_ideation_phase) }
+  describe 'in an active ideation phase without custom fields' do
+    let(:project) { create(:project_with_active_ideation_phase, p) }
```

Comment:
I'm not really following what `p` is here.

Reply:
typo! - don't know why it didn't break something though